lib/taiko.js

```diff
@@ -1885,6 +1893,7 @@ function dropDown(attrValuePairs, ...args) {
             const options = setNavigationOptions({});
             await doActionAwaitingNavigation(options, async () => {
                 const result = await runtimeHandler.runtimeCallFunctionOn(selectBox, null, { nodeId: nodeId, arg: value });
+                if(result.result.value instanceof Object) throw new Error(result.result.value);
```

Comment:
Please be specific about the type you compare

For e.g.
```
if(result.result.value.stack instanceof [whatever the instance of e is]) ...
```
Make sure you do a proper null check to see if there's a message and stack

Also 
```
new Error(result.result.value) 
```

will throw the error as follows

```
   throw new Error({message: 'This is a message', 'stack': e});
   ^

Error: [object Object]
    at Object.<anonymous> (/Users/zabilcm/Projects/sample/sample.js:23:10)
```

As you can see it has gobbled up the error to fix this you will have to throw it as

```
throw result.result.value.stack
```

or 

```
throw new Error(result.result.message)
```

Is there some kind of test for this?